condor_watch_q.py

```diff
@@ -623,11 +661,26 @@ def display_temporary_message(msg, enabled=True):
         yield
 
 
-PROJECTION = ["ClusterId", "Owner", "UserLog", "JobBatchName", "Iwd"]
+PROJECTION = [
+    "ClusterId",
+    "Owner",
+    "UserLog",
+    "DAGManNodesLog",
+    "JobBatchName",
+    "Iwd",
+    "DAG_NodesTotal",
```

Comment:
Do we have the wiring for this, assuming it exists at the time of the initial query? (We can do the followup queries thing later).